lib/synapse_pay_rest/models/subnet/subnet.rb

```diff
@@ -184,20 +186,15 @@ def lock
     #   subnet_id [String]
     #   transaction_id [String]
     # }
-    def ship_card(fee_node_id, cardholder_name, **options)
+    def ship_card(fee_node_id, cardholder_name, shipping_speed, card_style_id, **options)
       payload = {
-        'fee_node_id' => fee_node_id,
-        'cardholder_name' => cardholder_name,
-        'expedite' => false
+          fee_node_id: fee_node_id,
+          cardholder_name: cardholder_name,
+          delivery: shipping_speed,
+          card_style_id: card_style_id
```

Comment:
no problem is these are sent with null values I assume?

Reply:
the API will return an error where appropriate